homeassistant/components/smarty/sensor.py

```diff
@@ -77,6 +81,21 @@ class SmartySensorDescription(SensorEntityDescription):
         device_class=SensorDeviceClass.TIMESTAMP,
         value_fn=get_filter_days_left,
     ),
+    SmartySensorDescription(
+        key="system_state",
+        translation_key="system_state",
+        value_fn=lambda smarty: smarty.system_state,
+    ),
+    SmartySensorDescription(
+        key="software_version",
+        translation_key="software_version",
+        value_fn=lambda smarty: smarty.get_software_version(),
+    ),
```

Comment:
We can define `sw_version` in the `device_info` so it will show on the device info page